src/components/controls/TextArea/TextArea.tsx

```diff
@@ -73,6 +75,13 @@ export const TextArea = React.forwardRef<HTMLSpanElement, TextAreaProps>(functio
     const isClearControlVisible = Boolean(hasClear && !disabled && inputValue);
     const id = originalId || innerId;
 
+    const ariaDescribedBy = useDescribedBy({
+        ariaDescribedBy: controlProps?.['aria-describedby'],
+        note,
+        error,
+        controlId: id,
+    });
```

Comment:
Why should ids of error message and note depend on controlId?

```suggestion
    const errorMessageId = useUniqId();
    const noteId = useUniqId();
    const ariaDescribedBy = [
        controlProps?.['aria-describedby'],
        note ? noteId : undefined,
        isErrorMsgVisible ? errorMessageId : undefined
    ].filter(Boolean).join(' ');
```